### PR TITLE
doc(melange): remove docs

### PR DESCRIPTION
**Pull request by jchavarri**
Temporarily remove docs from public site until Melange integration is supported properly.

### CODE REVIEW

**Review by Alizter — APPROVED** [review]

**Comment by rgrinberg** [comment]
@jchavarri you can also just keep the docs and just remove them from the index? That would achieve the same effect.

**Comment by jchavarri** [comment]
@rgrinberg I believe that would remove the page from the index, but it'd still be accessible by typing it in the browser url directly.

**Comment by rgrinberg** [comment]
Yes, I think that would be acceptable. In any case, do as you prefer. It's just a temporary solution anyway.

Rudi.

File: doc/dune-files.rst [hunk 1]
```diff
@@ -586,10 +586,9 @@ to use the :ref:`include_subdirs` stanza.
 
 - ``(modes <modes>)`` is for modes which should be built by default. The
   most common use for this feature is to disable native compilation
-  when writing libraries for the OCaml toplevel. It can also be used to define
-  libraries that should be compiled with Melange (see :ref:`melange_main`). The
-  following modes are available: ``byte``, ``native``, ``best``, and ``melange``.
-  ``best`` is ``native`` or ``byte`` when native compilation isn't available.
+  when writing libraries for the OCaml toplevel. The following modes are
+  available: ``byte``, ``native`` and ``best``. ``best`` is ``native`` or
```
Comment (christinerose) [inline doc/dune-files.rst:590]: For some reason, it's not allowing me to use the suggest mode. 

This just needs a comma after `native` in the first sentence as part of the series: `byte`, `native`, and `best`.

**Review by christinerose — COMMENTED** [review]
One minor grammar suggestions. Looks great!

File: doc/dune-files.rst (hunk 1 above)
Comment (christinerose) [reply doc/dune-files.rst:590]: (Oh it's already merged! That's why I couldn't use the suggest mode!)

**Pull request merged**